apps/modernization-ui/src/components/Table/RangeToggle/RangeToggle.tsx

```diff
@@ -7,7 +7,7 @@ import { usePageMaybe } from 'page';
 import { Select } from '@trussworks/react-uswds';
 import styles from './range-toggle.module.scss';
 
-type SupportedContext = 'pages' | 'conditions' | 'questions' | 'valuesets';
+export type SupportedContext = 'pages' | 'conditions' | 'questions' | 'valuesets' | 'templates' | 'businessRules';
```

Comment:
Issue: we don't want to keep adding to these contexts. This just keeps growing and growing.